src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementServiceTest.java

```diff
@@ -2767,7 +2766,8 @@ void should_succeed() {
 
                 taskManagementService.terminateTask(taskId, terminateInfo);
 
-                assertEquals(CFTTaskState.COMPLETED, taskResource.getState());
+                assertEquals(CFTTaskState.TERMINATED, taskResource.getState());
```

Comment:
Just to be sure I've a question. 
We use taskState "TERMINATED" for the following states and hold them terminationReason?
TERMINATED, CANCELLED, COMPLETED and

Reply:
yes that is correct all tasks should end up in terminated state with the reason being stored in the terminatereason